lib/ansible/modules/cloud/vmware/vmware_host_firewall_manager.py

```diff
@@ -44,7 +44,6 @@
     - Each member of list is rule set name and state to be set the rule.
     - Both rule name and rule state are required parameters.
     - Please see examples for more information.
-    default: []
```

Comment:
I am not sure the reason behind removing this default value.

Reply:
Please see my commit message. This doesn't work with Atom. However, `default: {}` would work..